libs/cli/src/generators/ui/schema.d.ts

```diff
@@ -1,6 +1,5 @@
 export interface HlmUIGeneratorSchema {
 	name?: string;
-	directory?: string;
```

Comment:
I don't think it is a good idea to remove the directory from the schema. 
If someone uses the directory param in the CLI this would not work anymore.
I would rather remove the x-prompt and react if the directory is not set.
Also i would then just set the options.directory from the config.

Reply:
That's right! I'll add this back, just without the prompt :)